src/components/Curations/Curations.js

```diff
@@ -0,0 +1,34 @@
+import styles from './Curations.css'
+import curationsData from '/src/mock/curations.json'
+import Link from 'next/link'
+import Image from 'next/image'
+
+export default function Curations() {
+    return (
+        <div className="dashboard-container">
+            {curationsData.map((curation) => (
+                <div className="curation-box" key={curation.id}>
+                    <h3 className="curation-title">{curation.title}</h3>
+                        <div className="contents-box">
+                            {curation.contents.map((content) => (
+                                content.type == "youtube" ? // content.type이 youtube이면 play/${id}로 라우팅, writing이면 url 이동
```

Comment:
```suggestion
                                isPlayable(content) ? // content.type이 youtube이면 play/${id}로 라우팅, writing이면 url 이동
```

회의 때 말한 것처럼 isPlayable 같은 함수를 전역으로 export 해서 사용하면, 이런 데에도 같은 로직을 이용할 수 있을듯!

Reply:
src/utils에 isPlayable.js로 함수 생성했는데, 이렇게 하면 되는지 확인 부탁드립니다!

Reply:
굿굿! 요거 승현이랑도 얘기해서 승현이 부분에서도 쓸 수 있게 해보자